integration-tests/test/test_propose.py

```diff
@@ -34,7 +34,7 @@
     "contract_2.rho": 197,
     "contract_3.rho": 329,
     "contract_4.rho": 782,
-    "contract_5.rho": 3892,
+    "contract_5.rho": 3868,
```

Comment:
Do you have some explanation why cost changed? Its. fine I believe, just would be good to know why it happened.